lib/clean-host.js

```diff
@@ -1,7 +1,19 @@
 
-var URL = require('url');
```

Comment:
Maybe the intent would be clearer to rather test if the global `URL` is set and require it otherwise?

Or we could also rely directly on it since it is available since `node@10`.

What do you think?

Reply:
That makes sense - you'd probably know better than me if there are folks depending on this using ancient node versions, but as long as this came in a major bump IMO it'd be simplest to just use the global URL since it is available in all [maintenance+](https://nodejs.org/en/about/previous-releases) releases of node